test/collatz.cl

```diff
@@ -2,6 +2,7 @@
 
   ;; set language standard
   (include *standard-cl-22*)
+  (include test-inc.clsp)
```

Comment:
It's there to be a missing include so it can be corrected by the selenium test.
The test hovers the squiggle under test-inc.clsp and then triggers the code action that selects the directory of the include file.
I can add more documentation.

Reply:
Honest truth is i'm still building out tests and such.  The selenium tests down below are basic smoke tests for the functionality to run itself, but the feature itself is large.  This part wasn't even planned initially, but people didn't like the idea of having to edit json to have working include paths, so i devised what i hope is the simplest way of doing it.

The short answer is it needs more but there was pressure to release the lsp and i'm trying to release a refresh that corrects a lot of bugs and makes things better.  I don't have tests that verify everything very well at this point... it is in better shape than when it was initially released.  My hope was that i could incrementally continue improving it without being gated as much as i have been, but I've also tried to deliver complete prs that have full functionality and it's a challenge to break up something this big when there are competing interests on it.

I've been tying to honor all the various desires people have over it, but maybe the right thing to do is just start from zero and write it again in PRs so that reviewers can follow it step by step.